packages/lesswrong/lib/collections/multiDocuments/schema.ts

```diff
@@ -78,25 +99,31 @@ const schema: SchemaType<"MultiDocuments"> = {
     canUpdate: ['members'],
     canCreate: ['members'],
     nullable: false,
-    label: "Summary Title",
     order: 10,
   },
   tabSubtitle: {
     type: String,
     canRead: ['guests'],
     canUpdate: ['members'],
+    canCreate: ['members'],
+    hidden: ({ formProps, document }) => !formProps?.lensForm && document?.fieldName !== 'description',
     optional: true,
     nullable: true,
     order: 20,
   },
   userId: {
     type: String,
     canRead: ['guests'],
+    canCreate: ['members'],
```

Comment:
I believe in some other collections `userId` gets filled in with the ID of the creating user with an `onCreate` callback, which prevents users from creating content that they attribute to someone else.

Reply:
I see this happening in tags but not in e.g. posts/comments.  But it does seem safer to ensure it that way, so I'll update it.